tests/data/placebo/test_trusted_advisor_status_filter/support.DescribeTrustedAdvisorChecks_1.json

```diff
@@ -0,0 +1,6399 @@
+{
+    "status_code": 200,
+    "data": {
+        "checks": [
+            {
+                "id": "rSs93HQwa1",
```

Comment:
this api call could use some manual trimming to get just the check in question.

Reply:
I can trim this down if you think it is worthwhile to keep this PR open and implement the filter based on my response below.